metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -949,9 +949,11 @@ def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
         ]
 
     def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
-        database_seen = set()
         databases, tables = self.get_all_databases_and_tables()
 
+        for _, database in databases.items():
+            yield from self.gen_database_containers(database)
```

Comment:
I would suggest to check whether the database is allowed and if allowed then only generate container for it.

if self.source_config.database_pattern.allowed(
                database_name
             ): 
        yield container

Reply:
This check is already performed by `get_all_databases_and_tables` function:
https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/aws/glue.py#L696
We iterate over the output of this function.
Are there any more concerns stopping us from merging this PR?